homeassistant/components/recorder/websocket_api.py

```diff
@@ -72,3 +73,30 @@ def ws_update_statistics_metadata(
         msg["statistic_id"], msg["unit_of_measurement"]
     )
     connection.send_result(msg["id"])
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "recorder/info",
+    }
+)
+@callback
+def ws_status(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict
+) -> None:
+    """Return status of the recorder."""
+    instance = hass.data[DATA_INSTANCE]
+
+    backlog = instance.queue.qsize() if instance and instance.queue else None
+    migration_in_progress = instance.migration_in_progress if instance else False
+    recording = instance._event_listener is not None if instance else False
```

Comment:
Let's add a `recording` property to the `Recorder` class and use that here

Reply:
meh

Reply:
Fixed in e1512a2